test/e2e/kubectl.go

```diff
@@ -1695,6 +1695,50 @@ metadata:
 			}
 		})
 	})
+
+	framework.KubeDescribe("Kubectl v1.6 deployment reaper", func() {
+		var nsFlag string
+		var name string
+
+		BeforeEach(func() {
+			nsFlag = fmt.Sprintf("--namespace=%v", ns)
+			name = "e2e-test-nginx-deployment"
+		})
+
+		AfterEach(func() {
+			By("deleting the deployment " + name)
+			framework.RunKubectlOrDie("delete", "deployment", name, nsFlag)
+
+			By("verifying replicasets were deleted")
```

Comment:
I am not sure whether rs deletion should be put here (in accord with usage of `AfterEach`), or it should be at `It(...)`.